src/Components/Results/Results.tsx

```diff
@@ -98,7 +98,8 @@ const Results = ({ type }: ResultsProps) => {
   const { formData } = useContext(Context);
   const { whiteLabel, uuid, programId, energyCalculatorRebateType } = useParams();
   const is211Co = formData.immutableReferrer === '211co';
-
+  const is211Nc = formData.immutableReferrer === '211nc';
```

Comment:
Can you add a feature flag for showing removing the help button.

On the backend it would look like this for NC:
```py
"featureFlags": {"default": [], "211nc": ["no_results_more_help"]},
```

Then on the FE it would look like:
```suggestion
  const noHelpButton = getReferrer('featureFlags').includes('no_results_more_help');
```

This will allow us to more easily implement this in the future.

Reply:
change